### ISSUE TITLE
Merge Sort

### ISSUE THREAD

**Issue by Avn007** [hacktoberfest2k20/DataStructures-and-Algorithms#88]

**Comment by Avn007** [comment]
Hey, can I work on this issue?

**Comment by s-ayush2903** [comment]
@Avn007 Please first propose in which language you plan implementing this

**Comment by Avn007** [comment]
I will be using C++ to solve the problem.

**Comment by DevanshuAnand** [comment]
hey! is this issue open ?.. I would like to work on it.

**Comment by abhinav-bhardwaj** [comment]
Hey, can I work on this using python?

**Comment by s-ayush2903** [comment]
> Hey, can I work on this using python?

@abhinav-bhardwaj Yes please, go ahead and create an issue for merge sort to be in python and then open a PR
